[no squash] Fix handling of skinned meshes for nodes (#16112)
luanti-org/luanti: doc/lua_api.md

    mesh = "",
    -- File name of mesh when using "mesh" drawtype
    -- The center of the node is the model origin.

>     -- Due to historical reasons, .obj files use a scale of 1 node = 1 unit;

Reviewer: ```suggestion
    -- For legacy reasons, models in OBJ format use a scale of 1 node = 1 unit;
```
